.github/workflows/release.yml

```
@@ -0,0 +1,43 @@
+name: Release
+
+on:
+  workflow_call:
+    inputs:
+      database:
+        required: true
+        type: string
+    secrets: inherit
+
+  workflow_dispatch:
+    inputs:
+      database:
+        description: "Database to push to"
+        required: true 
+        default: 'Staging'
+        type: choice
+        options:
+          - Staging
+          - Production
+
+jobs:
+  test-and-release:
+    name: Push notebooks to database
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Python 3.11
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.11
+
+      - name: Install release script
+        run: |
+          pip install git+https://github.com/frankharkins/learning-content-tools.git#subdirectory=iql-lesson-sync
+
+      - name: Publish to database
+        env:
+          DIRECTUS_DATABASE: ${{ inputs.database }}
+          DIRECTUS_TOKEN: ${{ secrets.DIRECTUS_TOKEN_${{ inputs.database }}
```

```suggestion
          LEARNING_API_TOKEN: ${{ secrets.DIRECTUS_TOKEN_${{ inputs.database }}
```

Should be like this?

---

Also replace the `secrets.DIRECTUS_TOKEN_` with `secrets.LEARNING_API_`?

---

Whoops, yes – thanks!